components/pipeline-service/staging/base/chains-signing-secrets.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: tekton-chains-signing-secret
+  annotations:
+    argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
+    argocd.argoproj.io/sync-wave: "-1"
```

Comment:
That is a good point! I think this ExternalSecret should be in "openshift-pipelines" namespace

Reply:
the namespace may be set automagically in some other yaml spot @hugares but @Roming22 can confirm

In case my earlier point was not clear, if it is namespace scoped, I'm also worried that with this particular sync wave setting, is it getting created before the namespace is ..... @enarha made a recent fix to make sure that did not happen, but @Roming22 should cross ref with whatever sync-wave was used there

Reply:
It seems ArgoCD is smart enough to deploy resources in the same sync-wave in the appropriate order.

Quoting @enarha comment in the JIRA issue:
>The issue seems to be that all external secrets in Konflux are created at (ArgoCD) sync-wave "-1", where almost all other Pipeline Service components are created at sync-wave "0" or higher.
https://github.com/openshift-pipelines/pipeline-service/pull/918 fixes the issue creating the namespaces at sync-wave "-1", given that at equal level, namespaces are created before any other RRs, that should be enough.

So my understanding is that we're fine.

Reply:
ok thanks for confirming @Roming22